src/reprepro_updater/helpers.py

```diff
@@ -157,8 +162,10 @@ def run_cleanup(repo_dir, rosdistro, distro, arch, commit):
     lockfile = os.path.join(repo_dir, 'lock')
     with LockContext(lockfile) as lock_c:
 
-        _clear_ros_distro(repo_dir, rosdistro, distro, arch, commit)
-        delete_unreferenced(repo_dir, commit)
+        if not _clear_ros_distro(repo_dir, rosdistro, distro, arch, commit):
+            raise RuntimeError('command failed')
```

Comment:
These look good however we should have unique descriptive outputs for the three different failures.